env

```diff
@@ -1,3 +1,4 @@
+FINSCHIA_VERSION="v2.0.1"
 FINSCHIA_SDK_VERSION="v0.48.1"
 WASMD_VERSION="v0.2.0"
 OSTRACON_VERSION="v1.1.2"
```

Comment:
When are this versions updated?

Reply:
Thank you for your comment. The new version is PR here, but it references cometBFT.
So, close it once, delete the automatically generated branch, and wait for it to be automatically generated the next day.
https://github.com/Finschia/finschia-wasm/pull/22